base/exports.jl

```diff
@@ -1197,6 +1197,9 @@ export
     init_worker,
     interrupt,
     isready,
+    IC_MASTER_SLAVE,
+    IC_ALL_TO_ALL,
+    IC_CUSTOM,
```

Comment:
Sort of defeats the purpose of enums, no? We could still unexport it and require usage to be `Multiprocessing.IC_MASTER_SLAVE` or `Parallel.IC_MASTER_SLAVE` once https://github.com/JuliaLang/julia/pull/11638 is merged. 

`interconnect` is fine.